cpp/ql/src/utils/modelgenerator/GenerateFlowModel.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
```

Comment:
Does this script lives in this location for all languages for which we do model generation?

Reply:
Yep. It's in the same spot for Java, C#, and Rust.